amd/src/quizgame.js

```diff
@@ -60,8 +60,14 @@ define(['jquery','core/yui', 'core/notification', 'core/ajax'], function($, Y, n
     var currentPointsLeft = 0;
     var context;
     var inFullscreen = 0;
+
     $('#mod_quizgame_fullscreen_button').on('click', function () {
-        fullscreen();
+        if (inFullscreen === 1) {
+            inFullscreen--;
```

Comment:
Hmmm I've completely forgotten why this ever was a number instead of just boolean, maybe we could change this